client/app/bundles/course/announcements/pages/AnnouncementsIndex/index.tsx

```diff
@@ -69,7 +71,7 @@ const AnnouncementsIndex = (): JSX.Element => {
           />
         )
       }
-      title={t(translations.header)}
+      title={announcementTitle ?? t(translations.header)}
```

Comment:
`videoTitle || t(translations.header)` in other places, why use `??` here?

Reply:
sorry for the oversight. It should be `videoTitle ?? t(translations.header)`